<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTransitionProperties</class>
 <widget class="QDialog" name="DlgTransitionProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>537</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Transition properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Transition type :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="TransitionTypeCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select a transition family</string>
       </property>
       <item>
        <property name="text">
         <string>None and basic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Zoom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slide</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Push</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luma-Bar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luma-Box</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luma-Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luma-Checker</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luma-Clock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Luma-Snake</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Duration :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="TransitionDurationCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Define the animation duration</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">0.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>seconde(s)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="TransitionTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">0</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBT">
       <property name="toolTip">
        <string>Accept modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../../RSCffDiaporama.qrc">
         <normaloff>:/img/action_ok.png</normaloff>:/img/action_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseBT">
       <property name="toolTip">
        <string>Cancel modification and exit dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../RSCffDiaporama.qrc">
         <normaloff>:/img/action_cancel.png</normaloff>:/img/action_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HelpBT">
       <property name="toolTip">
        <string>Open help page describing this dialog box</string>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../../RSCffDiaporama.qrc">
         <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../RSCffDiaporama.qrc"/>
  <include location="../../RSCffDiaporama.qrc"/>
 </resources>
 <connections/>
</ui>
